### ISSUE TITLE
Copying a workflow ignores translations 

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/Panoptes-Front-End#4733]
## Expected behavior
If I create a new workflow from an existing workflow, any translations for that workflow should be copied to the new workflow

## Current behavior
The copy workflow button calls `api.type('workflows').create()` which will create a new resource with no translations. There's no way for Panoptes to know that this is a copy of an existing workflow, and copy the translation resources.

## Steps to replicate
Copy a workflow that is available in more than one language.

**Comment by camallen** [comment]
how will we address this? I assume copying via the API will be the optimal solution (vs client side).

**Comment by eatyourgreens** [comment]
Copying via the API sounds good to me. The project builder would need to be updated so that it isn't just creating a new workflow resource, but properly copies the workflow via the API.

**Comment by camallen** [comment]
I believe that copying workflows is available via the API as described in https://github.com/zooniverse/Panoptes/issues/2031#issuecomment-455591241

`POST /projects/:project_id/links/workflows` be careful not to PUT as mentioned in the linked issue.

that will need features added to copy the translations and other media resources over as well. I couldn't quickly find where the lab handles the linked workflow resource cloning, any advice?

**Comment by eatyourgreens** [comment]
I think it all happens here, with `workflowData` being passed in as a copy of the original workflow data.
https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/lab/actions/workflow.js

**Comment by eatyourgreens** [comment]
Yeah, the workflow creation form is passed a `workflowToClone` prop, so the code's a bit hard to follow.
https://github.com/zooniverse/Panoptes-Front-End/blob/e60b3cc7e0deb94e124dfc0c58777127069a03aa/app/pages/lab/workflow.cjsx#L92-L95

**Comment by eatyourgreens** [comment]
Not so fast, stalebot!

**Comment by srallen** [comment]
This is the likely cause of https://github.com/zooniverse/Panoptes-Front-End/issues/5822

**Comment by eatyourgreens** [comment]
Closed by #6212.
